tools/format_ninja.py

```diff
@@ -56,6 +56,11 @@ def remove_directory_path(path: str, directory: str) -> str:
 
 
 def normalize_if_pathlike(path: str, directory: str) -> str:
+
```

Comment:
nit empty line

Reply:
i think the pre-commit adds this, if the pre-commit checks fails will remove this.